Cargo.lock

```diff
@@ -115,15 +115,15 @@ dependencies = [
 
 [[package]]
 name = "anstyle"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
Seems you have accidentally bumped other dependencies in this PR. Would you please revert these changes?